### PR TITLE
 logout option provided in the page that comes when some user who does n...

### PR DESCRIPTION
**Pull request by lahiru823**
Fix for - Jira#APPM-224
(There is no logout option provided in the page that comes when some user who does not have publsiher role login to publisher)

### CODE REVIEW

File: modules/apps/store/extensions/assets/webapp/pages/subscriptions.jag [hunk 1]
```diff
@@ -33,6 +33,13 @@ require('/modules/store.js').exec(function (ctx) {
 
 	var subscriptions = [];
 	var userData = authenticator.getLoggedInUser();
+
+	//If the user is not logged in, redirect to the login page
+	if (userData == null) {
+		response.sendRedirect('/store');
```
Comment (rushmin) [inline modules/apps/store/extensions/assets/webapp/pages/subscriptions.jag]: Spacing issue.

File: modules/apps/store/apis/eventpublish.jag [hunk 2]
```diff
@@ -21,7 +21,11 @@
     var server = require('store').server;
     var user = server.current(session);
     if (!user) {
-        throw "user is not logged in";
+        //Capturing the UI activity change only if user is logged in
+        if (log.isDebugEnabled()) {
+            log.debug("user is not logged in");
```
Comment (dinusha-dilrukshi) [inline modules/apps/store/apis/eventpublish.jag]: Capitalize sentence.

Comment (lahiru823) [inline modules/apps/store/apis/eventpublish.jag]: fixed with commit# https://github.com/lahiru823/carbon-appmgt/commit/5923f4ea7d0161e93331bdff0f56e1ad3a8cb578

File: modules/apps/store/extensions/assets/webapp/pages/subscriptions.jag (hunk 1 above)
Comment (lahiru823) [inline modules/apps/store/extensions/assets/webapp/pages/subscriptions.jag]: fixed with commit# https://github.com/lahiru823/carbon-appmgt/commit/490a3c4048142182002a4cec49a15ef12d1de5c5

**Pull request merged**